### ISSUE TITLE
Insert new row in regression

### ISSUE THREAD

**Issue by squidink7** [numworks/epsilon#1824]
#### Problem you'd like to fix
I've often been laying out a data table in regression, and I sometimes when editing a row make a mistake and click the delete button one too many times, deleting a row and offsetting all the subsequent rows causing me to go change them all manually.

It would be nice if you could insert a row below the one selected with something like shift enter, which could help amend small mistakes tremendously, especially with large datasets
![image](https://user-images.githubusercontent.com/56238661/113841287-b6280580-97d0-11eb-9d0b-c32de9b9a2fc.png)

(screenshot from the custom Omega firmware, but confirmed to be happening on stock epsilon too, and I thought I'd make the report upstream to help as many people as possible)

#### Describe the solution you'd like
Some key combination for inserting a blank row beneath the currently selected one.

#### Describe alternatives you've considered
Leaving as is would still work, as you can add data at the bottom then later sort the data by selecting the column header, but for data sorted not by numerical value this can be a challenge keeping everything organised.

**Comment by Stephan-P** [comment]
You can add the extra line at the bottom of the list without any problem and the regression will take care of it perfectly.
![image](https://user-images.githubusercontent.com/10008599/113843729-22384880-9794-11eb-8bff-ab0b480fb4c9.png)
![image](https://user-images.githubusercontent.com/10008599/113843786-30866480-9794-11eb-97cc-27c946bdf4c5.png)

**Comment by squidink7** [comment]
Indeed, and this is fine in most cases, it would just help when initially setting out the data in the table. For example some problems I had to solve in school contained large data sets in a table where X was not linear so matching each Y value was not very easy and the order the data was presented in mattered greatly, and copying over the data became rather difficult when the table on my calculator was out of order compared to the presented data in the textbook, and I had to search for the matching entries rather than inputting them one after the other.
![image](https://user-images.githubusercontent.com/56238661/113856302-32761500-97e0-11eb-9d1c-e5d5ed64a197.png)
![image](https://user-images.githubusercontent.com/56238661/113856530-87199000-97e0-11eb-9412-cda1adafe96a.png)

**Comment by Stephan-P** [comment]
I guess what you want (and need) is the option to insert a full row, but also a single cell inside a column. The latter would help with inserting a value that was skipped upon entry.
- Insert Full row/Single value

**Comment by artaxxx** [comment]
Thanks @squidink7  ! Yes, indeed, it could be useful! But I feel this one is not a priority
